lib/iterable_basics.dart

```diff
@@ -141,6 +141,11 @@ extension IterableBasics<E> on Iterable<E> {
   num sum(num Function(E) addend) => this.isEmpty
       ? 0
       : this.fold(0, (prev, element) => prev + addend(element));
+
+  /// Returns a random element of [this], or [null] if [this] is empty.
```

Comment:
Can you include a comment about what `seed` is?

Something like: if [seed] is not null, it will be used to determine the internal state of [Random]. See [math.Random].

Reply:
Done.